clayui.com/gatsby/onCreateNode.js

```diff
@@ -220,6 +221,12 @@ module.exports = exports.onCreateNode = ({actions, getNode, node}) => {
 			value: sibling || '',
 		});
 
+		createNodeField({
+			name: 'siblings',
+			node,
+			value: siblings || '',
+		});
```

Comment:
Default value should probably be an array since that is what we expect from siblings